kahuna/public/js/components/gu-lazy-gallery/gu-lazy-gallery.html

```diff
@@ -0,0 +1,60 @@
+<div class="gallery" gu:lazy-gallery-shortcuts>
+
+    <div class="gallery-item" ng-if="galleryCtrl.item">
+
+        <div class="gallery-item__select gallery-item__select--selected"
+             ng:if="ctrl.imageHasBeenSelected(galleryCtrl.item)">
+            <input type="checkbox"
+                   class="gallery-item__select__checkbox"
+                   id="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--select"
+                   checked="true"
+                   ng:click="ctrl.deselect(galleryCtrl.item)"/>
+            <label for="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--select"
+                   class="gallery-item__select__checkbox__label">
+                <gr-icon>check_box</gr-icon>
+            </label>
+        </div>
+        <div class="gallery-item__select"
+             ng:if="!ctrl.imageHasBeenSelected(galleryCtrl.item)"
+             ng:class="{ 'gallery-item__select--no-pointer-events': ctrl.inSelectionMode}">
+            <input type="checkbox"
+                   class="gallery-item__select__checkbox"
+                   id="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--deselect"
+                   ng:click="ctrl.select(galleryCtrl.item)" />
+            <label for="id-{{::galleryCtrl.item.data.id}}-gallery-item__select__checkbox--deselect"
+                   class="gallery-item__select__checkbox__label">
+                <gr-icon>check_box</gr-icon>
+            </label>
+        </div>
+
+        <div class="gallery-item__preview"
```

Comment:
Not sure if you meant Mixpanel tracking @akash1810 but we no longer have a valid MP account, so don't do that @ShaunYearStrong  ^_^